wagtail_ab_testing/templates/wagtail_ab_testing/add_form.html

```diff
@@ -11,12 +11,16 @@
 
         <section class="section">
             <div class="section__title">
+                <div class="section__icon-container">
+                    {% icon "arrow-down" classname="section__icon" %}
+                </div>
                 <h3>{% trans "Enter test details" %}</h3>
             </div>
 
             <div class="section__content nice-padding">
                 <ul class="fields">
                     <li>
+                        {# TODO: Wagtail 6.0 introduced {% formattedfield %} to replace this template include #}
```

Comment:
Do we have a ticket to cover this?

Reply:
No, this is just a general note to future maintainers that they should replace this code once support for Wagtail 5.2 and older is dropped.

I don't think a ticket to go along with this note is necessary.